packages/peregrine/lib/Toasts/useToastContext.js

```diff
@@ -60,7 +60,8 @@ const reducer = (prevState = initialState, action = {}) => {
     }
 };
 
-const ToastContext = createContext();
+const ToastContext =
+    window.__ToastContext || (window.__ToastContext = createContext());
```

Comment:
Why is this necessary? I feel like you told me you worked this out in the past and didn't have to do this.